dev/com.ibm.ws.microprofile.config.1.1_fat/test-applications/cdiConfig.jar/src/com/ibm/ws/microprofile/appConfig/cdi/beans/ConfigFieldInjectionBean.java

```diff
@@ -203,25 +167,32 @@ public URL getOPTIONAL_MISSING_KEY() {
     }
 
     @Override
-    public Pizza getPIZZA_KEY() {
-        return PIZZA_EXISTING_PROP;
+    public String getMISSING_KEY() {
+        return MISSING_KEY_WITH_DEFAULT_VALUE;
     }
 
     @Override
-    public Pizza getPIZZA_MISSING_KEY() {
-        return PIZZA_MISSING_PROP;
+    public Pizza getPIZZA_GOOD_KEY() {
+        return PIZZA_GOOD_PROP;
     }
 
     @Override
-    public Pizza getPIZZA_GOOD_KEY() {
-        return PIZZA_GOOD_PROP;
+    public Parent getPARENT_KEY() {
+        return null;
     }
 
-//    public Car<Red> getRED_CAR_KEY() {
-//        return RED_CAR_PROP;
-//    }
-//
-//    public Car<Blue> getBLUE_CAR_KEY() {
-//        return BLUE_CAR_PROP;
-//    }
+    @Override
+    public Animal getANIMAL_KEY() {
+        return null;
+    }
+
+    @Override
+    public Pizza getPIZZA_KEY() {
+        return null;
+    }
+
+    @Override
+    public Pizza getPIZZA_MISSING_KEY() {
+        return null;
+    }
```

Comment:
These methods which return `null` look like they're not referenced anywhere any more?